test/tpu/pjrt_test_job.yaml

```diff
@@ -0,0 +1,44 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  generateName: pjrt-test-job-
+spec:
+  affinity:
+    nodeAffinity:
+      requiredDuringSchedulingIgnoredDuringExecution:
+        nodeSelectorTerms:
+        - matchExpressions:
+          - key: tpu.googleapis.com/name
+            operator: In
+            values:
+            # Update with your TPU name
+            - gke-tpu
+          - key: tpu.googleapis.com/type
+            operator: In
+            values:
+            - v3-8
+  restartPolicy: Never
+  volumes:
+  # Increase size of tmpfs /dev/shm to avoid OOM.
+  - name: dshm
+    emptyDir:
+      medium: Memory
+  containers:
+  - name: pjrt-test
+    securityContext:
+      privileged: true
+    image: gcr.io/$PROJECT_ID/pytorch-xla-test:$BUILD_ID
+    command:
+    - bash
+    - -c
+    - |
+      ls /root/anaconda3/envs/pytorch/lib/python3.8/site-packages/ | grep libtpu
+      sudo rm -rf /usr/local/lib/python3.8/dist-packages/libtpu*
+      sudo pip3 install torch_xla[tpuvm]
```

Comment:
You can skip the steps to install libtpu if `BUNDLE_LIBTPU=1` is set during the build

Reply:
Done